File: AppLaunchDemo/AppLaunchDemo/ViewController.m
//
//  ViewController.m
//  AppLaunchDemo
//
//  Created by zzyong on 2020/5/8.
//  Copyright © 2020 zzyong. All rights reserved.
//

#import "ViewController.h"
#import "MyClassA1.h"
#import <MyStaticLibA/MyView.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
    [self setupSubviews];
}

- (void)viewWillAppear:(BOOL)animated
{
    [super viewWillAppear:animated];
    [MyClassA1 sayHello];
}

- (void)viewDidAppear:(BOOL)animated
{
    [super viewDidAppear:animated];
    [self doSomething];
}

- (void)setupSubviews
{
    MyView *view = [[MyView alloc] init];
    view.frame = CGRectMake(100, 100, 100, 100);
    [self.view addSubview:view];
}

- (void)doSomething
{
    
}

@end

File: iRead/iRead/ViewController.swift
//
//  ViewController.swift
//  iRead
//
//  Created by zzyong on 2020/2/25.
//  Copyright © 2020 zzyong. All rights reserved.
//

import UIKit
import IRHexColor
import IRCommonLib

class ViewController: UIViewController {

    lazy var epubParser: FREpubParser = FREpubParser()
    lazy var bookCover: UIImageView = UIImageView()
    
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        bookCover.contentMode = .center
        self.view.addSubview(bookCover)
        
        guard let bookPath = Bundle.main.path(forResource: "支付战争", ofType: "epub") else { return}
        guard let book: FRBook = try? epubParser.readEpub(epubPath: bookPath) else { return }
        if let coverUrl = book.coverImage?.fullHref {
            bookCover.image = UIImage.init(contentsOfFile: coverUrl)
        }
    }
    
    override func viewDidLayoutSubviews() {
        super.viewDidLayoutSubviews()
        
        bookCover.frame = self.view.bounds
    }
}

